src/base/bittorrent/session.cpp

```diff
@@ -2354,22 +2355,34 @@ void Session::exportTorrentFile(TorrentHandle *const torrent, TorrentExportFolde
     Q_ASSERT(((folder == TorrentExportFolder::Regular) && !torrentExportDirectory().isEmpty()) ||
              ((folder == TorrentExportFolder::Finished) && !finishedTorrentExportDirectory().isEmpty()));
 
+    using namespace Utils::DB;
+    QSqlQuery query;
```

Comment:
@sledgehammer999, @glassez:  

Perhaps you should make a new class to manage SQLite read/write operations? 

Sort of like `SettingsStorage`?

Reply:
@LordNyriox I'll have to see fist, but I don't think the DB code (due to the nature of SQL statements) can be abstracted away in to a class.
Any abstraction will be done last, because we need to agree on a format of storing data in the db.